src/contexts/App.tsx

```diff
@@ -5,8 +5,7 @@ export interface AppProviderProps {
 }
 
 const initialState = {
-  container: HTMLDivElement,
```

Comment:
I had to do a double take on this line. `initialState` _looks_ like an `interface` definition, but you're actually creating an object who's `container` property is set to the `HTMLDivElement` interface. Later you set it to an element as expected.